test/integration/elasticquota_controller_test.go

```diff
@@ -116,9 +110,9 @@ func TestElasticController(t *testing.T) {
 		t,
 		testCtx,
 		true,
-		scheduler.WithFrameworkOutOfTreeRegistry(registry),
+		scheduler.WithFrameworkOutOfTreeRegistry(fwkruntime.Registry{coscheduling.Name: coscheduling.New}),
```

Comment:
Why do we init coscheduling other capacity scheduling in integration test of elasticquota?  This is the issue left before.

Reply:
it's a typo...